gen/definitions/bgp_peer_address_family.yaml

```diff
@@ -0,0 +1,99 @@
+---
+name: BGP Peer Address Family
+class_name: bgpPeerAf
+dn: sys/bgp/inst/dom-[%s]/peer-[%s]/af-[%s]
+ds_description: This data source can read the BGP peer address family configuration.
+res_description: This resource can manage the BGP peer address family configuration.
+doc_path: Routing%20and%20Forwarding/bgp:PeerAf/
+parents:
+  - BGP Peer Template
```

Comment:
I guess those would need to be updated, referring to the peer instead of the peer template..

Reply:
You re correct. Fixed.